src/routes/routeConstants.ts

```diff
@@ -13,43 +18,188 @@ export const COLONY_EXTENSIONS_ROUTE = `extensions`;
 export const COLONY_INTEGRATIONS_ROUTE = `integrations`;
 export const COLONY_INCORPORATION_ROUTE = `incorporation`;
 export const COLONY_ADVANCED_ROUTE = `advanced`;
-export const COLONY_EXTENSION_DETAILS_ROUTE = `extensions/:extensionId/*`;
-export const COLONY_EXTENSION_DETAILS_SETUP_ROUTE = `extensions/:extensionId/setup`;
-export const COLONY_FUNDING_ROUTE = `funds`;
+export const COLONY_EXTENSION_DETAILS_ROUTE = `${COLONY_EXTENSIONS_ROUTE}/:extensionId/*`;
+export const COLONY_EXTENSION_SETUP_ROUTE = `setup`;
+export const COLONY_EXTENSION_DETAILS_SETUP_ROUTE = `${COLONY_EXTENSIONS_ROUTE}/:extensionId/${COLONY_EXTENSION_SETUP_ROUTE}`;
+export const COLONY_BALANCE_ROUTE = `balance`;
```

Comment:
Apologies, but this should be `balances` I made a typo in my last feedback.

This route should be for the `/balances` page, not the Incoming funds page, which should be `/incoming`.

The `balances` page was added in this PR - https://github.com/JoinColony/colonyCDapp/pull/1174

- app.colony.io/meta/balances
    - Page Title: Balances | Colony - Metacolony
- http://app.colony.io/meta/incoming
    - Page Title: Incoming | Colony - Metacolony

![image](https://github.com/JoinColony/colonyCDapp/assets/33682027/05d15255-8409-4a96-8130-d9e5b1afbffb)